### ISSUE TITLE
Batch normalization layer

### ISSUE THREAD

**Issue by weizequan** [antingshen/resnet-protofiles#11]
In your *.prototxt, all batch norm layers;
batch_norm_param {
        use_global_stats: false
    }
But there is a detailed description http://caffe.berkeleyvision.org/tutorial/layers/batchnorm.html
"By default, it is set to false when the network is in the training phase and true when the network is in the testing phase."
Therefore, in your code, you set use_global_stats to false for both training and testing phase. Which is better?

**Comment by antingshen** [comment]
The `*_deploy.prototxt` files are the ones used for inference/testing, and they have `use_global_stats: true`, so this is the same as the default. Feel free to try removing it and see if it makes a difference during val, which I didn't think about. If you find a difference let me know.
